lib/node_modules/@stdlib/math/base/special/imuldw/test/fixtures/c/runner.c

```diff
@@ -0,0 +1,394 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+* Generate C test fixtures.
+*
+* ## Notes
+*
+* -   Run this script from the directory in which fixtures should be written.
+*
+*/
+#include <stdlib.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <math.h>
+#include <sys/time.h>
+
+/**
+* Computes the doubleword product of two words, signed.
+*
+* From: http://www.hackersdelight.org/hdcodetxt/muldws.c.txt
+*
+* @param w output array
+* @param u first word
+* @param v second word
+*/
+void muldws1( int w[], int u, int v ) {
```

Comment:
```suggestion
void muldws1( int *w, int u, int v ) {
```

For consistency elsewhere in this file.